exercises/queen-attack/src/queen_attack.h

```diff
@@ -0,0 +1,19 @@
+#ifndef QUEEN_ATTACK_H
+#define QUEEN_ATTACK_H
+
+#include <stdint.h>
+
+enum result {
+   CAN_NOT_ATTACK,
+   CAN_ATTACK,
+   INVALID_POSITION
+};
+
+typedef struct {
+   uint8_t row;
+   uint8_t column;
+} queen_position_t;
+
+uint8_t can_attack(queen_position_t queen_1, queen_position_t queen_2);
```

Comment:
The return value here should be the enum type (which I'd suggest renaming). Maybe `attack_status`?

Reply:
I agree with both 👍